app/src/main/java/de/tum/in/tumcampusapp/service/DownloadService.kt

```diff
@@ -227,6 +227,11 @@ class DownloadService : JobIntentService() {
                     Const.TOP_NEWS -> success = service.downloadTopNews()
                     else -> {
                         success = service.downloadAll(force)
+
+                        if (success) {
+                            Utils.setSetting(service, Const.EVERYTHING_SETUP, true)
```

Comment:
What exactly is hidden behind this flag? It would be great to document it somewhere, as I don't think to recall what exactly the idea was and I'm not sure it is used here correctly.

Reply:
It describes whether a full first launch of the app has occurred, including downloading all data. If this is not true, then `StartupActivity` displays the loading indicator, indicating a longer-than-normal loading.

Reply:
Why do we separately handle syncing the calendar below then?

Reply:
Especially why do we set this variable to true a second time in a if-branch that almost never will be executed?